templates/docs/examples/patterns/accordion/default.html

```diff
@@ -20,16 +22,20 @@
       </section>
     </li>
     <li class="p-accordion__group">
-      <button type="button" class="p-accordion__tab" id="tab2" aria-controls="tab2-section" aria-expanded="false">Networking</button>
-      <section class="p-accordion__panel" id="tab2-section" aria-hidden="true" aria-labelledby="tab2">
+      <div role="heading" aria-level="2" class="p-accordion__heading">
+        <button type="button" class="p-accordion__tab" id="tab4" aria-controls="tab4-section" aria-expanded="false">Networking</button>
```

Comment:
It's really a nitpicking, but this could be `tab2` and the last one `tab3` (especially that `aria-labelledby` say so).

Reply:
ah - good catch! fixing

Reply:
fixed